### DISCUSSION TITLE
Error installing with air 33.1.1.406

### DISCUSSION THREAD

**Discussion by raresn** [airsdk/Adobe-Runtime-Support#726] [Q&A] [answered]
Hey guys,
I have recently upgraded to air 33.1.1.406 (Windows).
The app compiles, but i get this when trying to deploy on iOS for testing:
WARNING: could not locate iTunesMetadata.plist in archive!
WARNING: could not locate Payload/On Time Agenda.app/SC_Info/On Time Agenda.sinf in archive!
ERROR: Install failed. Got error "ApplicationVerificationFailed" with code 0xe8008001: Failed to verify code signature of /private/var/installd/Library/Caches/com.apple.mobile.installd.staging/temp.PPdKy5/extracted/Payload/On Time Agenda.app : 0xe8008001 (An unknown error has occurred.)

Anyone had this problem before? (it worked fine with Air 33.1.345)

**Comment by ajwfrost** [comment, marked as answer]
Yes, please see #511 and there's a link there to a temporary adt.jar file that will fix this. We'll be pushing out an updated SDK with the fix shortly..

thanks

**Reply by raresn** [reply]
I honestly did not expect an answer this fast!
Simply stellar support!

Thank you, I will try it out!

**Reply by ajwfrost** [reply]
:-) the easier the answer, the quicker it comes... working out what the actual problem was - and then solving it - did take a few months!

**Comment by studiomaxer3d** [comment]
I have a similar issue but the adt.jar file didn't fix it for me.

**Reply by vree** [reply]
Same here.  I just tried replacing the adt.jar file in 33.1.1.406 , but I still get the same issue when trying to submit IPA:
ERROR ITMS-90179: "Invalid Code Signing. error...

Using Big Sur and Transporter.  I've updated my cert and provisional profiles just to make sure that was not. the issue.
